src/types/internal.ts

```diff
@@ -45,7 +47,15 @@ export type Day =
   | 'thursday'
   | 'friday'
   | 'saturday'
-  | 'sunday'
+  | 'sunday';
+
+export enum AppEnum {
+  TWINE_API = 'TWINE_API',
+  VISITOR = 'VISITOR_APP',
+  VOLUNTEER = 'VOLUNTEER_APP',
+  DASHBOARD = 'DASHBOARD_APP',
```

Comment:
I wonder whether it's worth calling this `DATA` rather than dashboard, then we're describing the subdomains. Or do you think thats too much coupling to hosting?

Reply:
Yeah, I think that's too much.